src/poetry/installation/executor.py

```diff
@@ -298,6 +299,18 @@ def _execute_operation(self, operation: Operation) -> None:
                 with self._lock:
                     trace = ExceptionTrace(e)
                     trace.render(io)
+                    if isinstance(e, ChefBuildError):
+                        name = operation.package.pretty_name
+                        io.write_line("")
+                        io.write_line(
+                            "<info>"
+                            "Note: This error originates from the build backend,"
+                            " and is likely not a problem with poetry"
+                            f" but with {name} not supporting PEP 517 builds."
+                            " You can verify this by running"
+                            f" 'pip wheel --use-pep517 {name}'."
```

Comment:
This should probably also include the concrete version number if there's any way we can get at it.